sdcm/provision/common/configuration_script.py

```diff
@@ -32,10 +32,12 @@ class ConfigurationScriptBuilder(AttrBuilder, metaclass=abc.ABCMeta):
     configure_sshd: bool = True
     disable_ssh_while_running: bool = False
     hostname: str = ''
+    legacy_rsa_keys: bool = True
 
     def to_string(self) -> str:
         script = self._start_script()
         script += self._script_body()
+        script += self._enable_legacy_rsa_keys()
```

Comment:
should this be called only for Ubuntu22? or it doesn't matter for other versions (including RHEL, Debian and old Ubuntu ones)?

Reply:
It shouldn't interfere with other versions, I think. It's necessary for any distro that uses the more recent version of openssh, so having this here should cover all such cases.

Reply:
did you test some other version or distros ?

Reply:
Would be happy to do so, waiting for available executors on Jenkins currently.

Reply:
Tested on other distros (previous ubuntu versions) and got some ssh failures, so I made it so that this line will only be called with Ubuntu22.04 for now.